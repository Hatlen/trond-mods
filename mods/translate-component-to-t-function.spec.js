import pluginTester from "babel-plugin-tester";
import babelI18nPlugin from "./translate-component-to-t-function";

pluginTester({
  plugin: babelI18nPlugin,
  babelOptions: {
    filename: ".babelrc",
    babelrc: true
  },
  snapshot: true,
  tests: [
    `
const C = () => (
  <div>
    <Translate
      count={props.totalVisitors}
      totalViews={numeral(props.totalVisitors).format("0,0")}
      i18nKey="analyze.components.overview_story.total_views"
    />
  </div>
);
    `.trim()
  ]
});
