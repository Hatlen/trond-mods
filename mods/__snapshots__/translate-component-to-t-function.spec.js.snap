// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Translate-to-withNamespaces-t 1. Translate-to-withNamespaces-t: 1. Translate-to-withNamespaces-t 1`] = `
"
const C = () => (
  <div>
    <Translate
      count={props.totalVisitors}
      totalViews={numeral(props.totalVisitors).format(\\"0,0\\")}
      i18nKey=\\"analyze.components.overview_story.total_views\\"
    />
  </div>
);

      ↓ ↓ ↓ ↓ ↓ ↓

\\"use strict\\";

var C = function C() {
  return React.createElement(\\"div\\", null, t(\\"analyze.components.overview_story.total_views\\", {
    count: props.totalVisitors,
    totalViews: numeral(props.totalVisitors).format(\\"0,0\\")
  }));
};
"
`;
